outlines/forms.md

```diff
@@ -0,0 +1,24 @@
+# Forms and Methods
+
+1. Methods
+  1. The concept of a method and how it corresponds to a form
```

Comment:
I think this would be a great place to elaborate more on the way a method works.

1. Simulations vs. (non-simulations? what are they called???)
2. The return value vs. the "updated" message, and which one triggers the callback
3. How methods are automatically retried
4. Meteor.Error vs. regular Error

I would prefer more detail on this bullet before "accepting" the outline.